src/kernel/chainparams.cpp

```diff
@@ -241,7 +241,7 @@ class CTestNetParams : public CChainParams {
         pchMessageStart[3] = 0x07;
         nDefaultPort = 18333;
         nPruneAfterHeight = 1000;
-        m_assumed_blockchain_size = 200;
+        m_assumed_blockchain_size = 215;
```

Comment:
Seems low, would expect 244.

```
$ du -csh --apparent-size testnet3/chainstate/ testnet3/blocks/
16G	testnet3/chainstate/
205G	testnet3/blocks/
221G	total
```

Reply:
For a node that's been offline for a year or so and then caught up yesterday:

```
16G	testnet3/chainstate/
195G	testnet3/blocks/
210G	total
```